modules/spo-binding-workloads.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-seccomp.adoc
+// * security/security_profiles_operator/spo-selinux.adoc
+
+ifeval::["{context}" == "spo-seccomp"]
+:seccomp:
+:type: seccomp
+:kind: SeccompProfile
+endif::[]
+ifeval::["{context}" == "spo-selinux"]
+:selinux:
+:type: SELinux
+:kind: SelinuxProfile
+endif::[]
+
+:_content-type: PROCEDURE
+[id="spo-binding-workloads_{context}"]
+= Binding workloads to profiles with ProfileBindings
+
+If you do not want to directly modify the `SecurityContext` of a `Pod` object, you can use the `ProfileBinding` resource to bind a security profile to the `SecurityContext` of a container.
+
+// Spell out why it's better to modify the binding over the SC.
+
+.Procedure
+
+. To bind a `Pod` that uses an `nginx:1.19.1` image to the `profile` example `seccomp` profile, create a `ProfileBinding` object in the same namespace with the `Pod` and the `{kind}` objects:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: security-profiles-operator.x-k8s.io/v1alpha1
+kind: ProfileBinding
+metadata:
+  name: nginx-binding
+spec:
+  profileRef:
+    kind: {kind}
+    name: profile
+  image: nginx:1.19.1
+----
+
+. Restart the pod to use the `ProfileBinding` object:
```

Comment:
hmm, we don't show the restarting of the pod here, is that implied? I'm not sure if this would be too much content, but what the binding does here is that without the binding the pod has no profile:
```
apiVersion: v1
kind: Pod
metadata:
  name: test-pod
  namespace: spo-test
spec:
  containers:
    - name: nginx
      image: nginxinc/nginx-unprivileged:1.21
```

We can verify this when we create the pod:
```
oc create -f pod01.yaml
oc get pod test-pod -o jsonpath='{.spec.containers[*].securityContext.seccompProfile}'
(nothing here)
```

Then we create the binding:
```
apiVersion: security-profiles-operator.x-k8s.io/v1alpha1
kind: ProfileBinding
metadata:
  name: nginx-binding
  namespace: spo-test
spec:
  profileRef:
    kind: SeccompProfile
    name: profile-complain-block-high-risk
  image: nginxinc/nginx-unprivileged:1.21
```

delete the pod and create it again:
```
oc delete pods test-pod
oc create -f pod01.yaml
```
and boom, it gets the profile as per the binding:
```
oc get pod test-pod -o jsonpath='{.spec.containers[*].securityContext.seccompProfile}'
{"localhostProfile":"operator/spo-test/profile-complain-block-high-risk.json","type":"Localhost"}%    
```